site/_scss/site/objects/_components.scss

```diff
@@ -0,0 +1,142 @@
+.grid.three {
+    display: grid;
+    grid-template-columns: 1fr;
+    row-gap: 20px;
+    column-gap: 20px;
+    @include media-breakpoint-up(md) {
+        grid-template-columns: 1fr 1fr 1fr;
+    }
+}
+.sub-page {
+    background-color: #fff;
+    h2 {
+        font-size: 22px;
+        margin: 1.5rem 0;
+    }
+    .section-header {
+        margin-top: 3rem;
+        font-weight: 600;
+        font-size: 20px;
+    }
+    .embed-responsive {
+        margin-bottom: 1rem;
+    }
+    .grid {
+        .col {
+            border: 1px solid #F2F2F2;
+            padding: 0px;
+            .icon {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                min-height: 140px;
+                img {
+                    max-width: 100%;
+                    &.podcast {
+                        max-height: 100px;
+                    }
+                }
+                &.twitter svg {
+                    margin-left: -25px;
+                }
+            }
+            h3 {
+                margin: 0px;
+                text-align: center;
+                font-size: 16px;
+            }
+            .content {
+                padding: 0px 20px 20px 20px;
+                p {
+                    font-size: 1rem;
+                    color: #777;
+                    &.title {
+                        font-weight: 500;
+                        margin: 0px;
+                    }
+                    &.strong {
+                        font-weight: 600;
+                    }
+                }
+                ul {
+                    padding-left: 20px;
+                    li {
+                        margin-bottom: 10px;
+                        color: #777;
+                        font-size: 1rem;
+                        line-height: 1.6em;
+                        list-style-image: url(/img/arrow.svg);
+                    }
+                }
+            }
+        }
+    }
+}
+.blog-card {
+    .post-thumbnail {
+        margin-bottom: 0px;
+        img {
+            transform: unset;
+            position: unset;
+        }
+    }
+    .card-body .post {
+        text-align: left;
+        a {
+            font-size: 16px;
+        }
+        .post-entry {
+            font-size: 16px;
+        }
+    }
+}
+.post-single-hero {
+    h1, h2, h3 {
+        text-align: left;
+    }
+    h1 {
+        margin-bottom: 2rem;
+        font-size: 2.25rem;
+    }
+}
+.getting-started {
+    h2, p {
+        color: #000 !important;
+    }
+    h2 {
+        font-size: 30px;
+        margin-bottom: 10px;
+    }
+    a.btn {
+        border: 1px solid #0095D3;
```

Comment:
@amymanion where possible, we should be defining using scss variables for hex values that are used more than once